integrators/integrators_body.hpp

```diff
@@ -14,80 +14,254 @@
 #include "integrators/symmetric_linear_multistep_integrator.hpp"
 #include "integrators/symplectic_runge_kutta_nyström_integrator.hpp"
 
-#define PRINCIPIA_CASE_SLMS(kind, method)                      \
-  case serialization::FixedStepSizeIntegrator::kind:           \
-    return SymmetricLinearMultistepIntegrator<methods::method, \
-                                              typename ODE::Position>()
+#define PRINCIPIA_CASE_SLMS(kind, method, action)      \
+  case serialization::FixedStepSizeIntegrator::kind: { \
+    action(method);                                    \
+  }
```

Comment:
if `action` does not return or break, this is a fallthrough. Maybe `break`? Perhaps even without a semicolon, so as to require a semicolon from the caller?

Reply:
Added a `break;`.  I hope that it won't be detected as dead code by some smart compiler.

Let's not play games with semicolons, it seems odd to force the macro to include a semicolon.